### DISCUSSION TITLE
HTML objects in PowerPoint slides

### DISCUSSION THREAD

**Discussion by rdatasculptor** [davidgohel/officer#510] [Ideas]
First if all: great package! I used to work with your very handy package ReporteRs some years ago, but officer definitely seems next level. 

I was wondering, will it be possible to be able to add html objects to PowerPoint slides, programmatically, using officer; and also add this content to the final .pptx file? I know in PowerPoint this is possible. Maybe I missed something, but it doesn't seem possible using officer. 

Just some fantasy I guess, but the ultimate goal for me would be to be able to add saved htmlwidgets to PowerPoint slides. Wouldn't it be wonderfull to give colleagues their familiar and loved PowerPoints together with all the nice interactive features that htmlwidgets can add to their presentations?

**Comment by davidgohel** [comment]
I don't think it's possible without an addin. You know it is possible? Can you share an example? 

I don't get the concept. HTML decks made with reveal for example are so good, why not just use them if you want html?

**Reply by rdatasculptor** [reply]
You may be right. Reveal should be my first choice as well regarding htmlwidgets, but users of the presentation need to be in control of additional texts, deleting slides, etc.

**Reply by davidgohel** [reply]
Can you share an example of a ppt with some html widget inside (you said you know it is possible) ?

> users... control etc

That's a catch 22, they will also want to not learn r python or js, they will want it for free and they will complain if it is not easy as 123. They want to edit html, they have to learn html. They want to drive a car, they have to learn how to drive a car... Very philosophical 😉

**Reply by rdatasculptor** [reply]
I will try to make an example, but I must admit I only read about it a while ago. If I can't make a real example, I will close this discussion 🙂

**Reply by davidgohel** [reply]
> I know in PowerPoint this is possible

**Reply by davidgohel** [reply]
There is nothing to close here. It stays as it is.

**Reply by rdatasculptor** [reply]
this is de information I was referring to https://www.youtube.com/watch?v=dea8XawH6no. I haven't made it work yet.

**Reply by davidgohel** [reply]
What the heck, I don't care about Ai generated content
Show the file result!

**Reply by davidgohel** [reply]
With htmlwidget

**Reply by davidgohel** [reply]
Actually don't
I am stopping arguing and helping there

**Reply by rdatasculptor** [reply]
Okay sorry for bothering you! Guess misunderstood the information out there. If I ever make it work, I will let you know
